zsh/configs/keybindings.zsh

```diff
@@ -10,4 +10,5 @@ bindkey "^R" history-incremental-search-backward
 bindkey "^P" history-search-backward
 bindkey "^Y" accept-and-hold
 bindkey "^N" insert-last-word
+bindkey "^B" push-line-or-edit
```

Comment:
`^B` also collides with the default tmux leader key. I found a [reference card](http://www.bash2zsh.com/zsh_refcard/refcard.pdf) (warning: PDF) that lists the default zle bindings in each mode (page13), which may be useful.